net-p2p/syncthing/syncthing-0.13.7-r1.ebuild

```diff
@@ -16,24 +16,32 @@ SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
 LICENSE="MPL-2.0"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
-IUSE=""
+IUSE="tools"
 
 DEPEND=""
 RDEPEND=""
 
+DOCS="README.md AUTHORS CONTRIBUTING.md"
+
 pkg_setup() {
 	enewgroup ${PN}
 	enewuser ${PN} -1 -1 /var/lib/${PN} ${PN}
 
-	# separate user for relaysrv
-	enewgroup ${PN}-relaysrv
-	enewuser ${PN}-relaysrv -1 -1 /var/lib/${PN}-relaysrv ${PN}-relaysrv
+	if use tools ; then
+		# separate user for relaysrv
+		enewgroup ${PN}-relaysrv
+		enewuser ${PN}-relaysrv -1 -1 /var/lib/${PN}-relaysrv ${PN}-relaysrv
+	fi
 }
 
 src_compile() {
 	export GOPATH="${S}:$(get_golibdir_gopath)"
 	cd src/${EGO_PN}
-	go run build.go -version "v${PV}" -no-upgrade || die "build failed"
+	if use tools ; then
+		go run build.go -version "v${PV}" -no-upgrade || die "build failed"
```

Comment:
I am here. Just got very busy for some time. I should have time for this tomorrow :-)